components/utils.ts

```diff
@@ -338,6 +338,7 @@ function updateUserProfile(
                 {},
             )
 
+            // ts-expect-error Legacy property.
```

Comment:
Intentionally no `@`, since we don't yet have `strict` set to true, it causes an error because this is valid code when strict is false.